### DISCUSSION TITLE
It seems some docs are inconsistent on how to install

### DISCUSSION THREAD

**Discussion by cpreston321** [nuxt/content#1065] [Ideas]
Some docs sites like nuxt3 give you example how to install with `npm`, `yarn` and  `pnpm`. The main [index](https://deploy-preview-1063--nuxt-content.netlify.app/) page should include all three options for the users since if someone is using different ways to install. I would be nice to see. Also if you go to getting started docs you have options for `yarn` & `npm` [here](https://deploy-preview-1063--nuxt-content.netlify.app/getting-started/installation)

**Comment by cpreston321** [comment]
I wanted to do a PR but it seems I don't have access to Docus yet so there was no styles for me :/ 

Anyways Thanks!

**Comment by lucpotage** [comment]
Problem was solved with the release. Now there is a video instead of the fake terminal window: https://content.nuxtjs.org/.
